asf_welsh_energy_consultation/pipeline/process_data.py

```diff
@@ -57,14 +63,37 @@ def get_enhanced_mcs():
         }
     )
 
-    return mcs
+    return df
+
+
+def get_total_cumsums(data, installation_date_col):
```

Comment:
This is merged from branch 10 which has already been reviewed

Reply:
just noticed the docstrings are missing the args info
```
Args:
   data:...
   installation_date_col:...
```